docs/configuration.md

```diff
@@ -753,7 +753,7 @@ interpretation of the options.
       no
     </td>
     <td>
-      Duration for which a signed URL should be valid.
+      Duration in minutes for which a signed URL should be valid.  When omitted, this defaults to 20 minutes.
```

Comment:
I don't see anything in the code that assumes this is in minutes. It appears this value is just parsed with https://golang.org/pkg/time/#ParseDuration